DependencyInjection/M6WebKafkaExtension.php

```diff
@@ -31,11 +32,14 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->loadProducers($container, $config);
         $this->loadConsumers($container, $config);
+
+        $container->setParameter('m6web_kafka.prefix_name', $config['prefix_services_name']);
     }
 
     /**
      * @param ContainerBuilder $container
-     * @param array            $config
+     * @param array $config
+     * @throws \Symfony\Component\DependencyInjection\Exception\BadMethodCallException
```

Comment:
IMHO this doc tag is not helpful if there's no `throw` statement in the method body, except if you explain in which case this exception is thrown (eg. `@throws \MyException If a wrong parameter is used`)